### PR TITLE
AP-878: Donations: "Other" tab

### PR DESCRIPTION
**Pull request by misicnenad**
## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

#### Desktop
- Stocks
image
image
image

- DAF
image

#### Mobile
- Stocks
image
image

- DAF
image

#### Desktop
- Stocks
image
image
- DAF
image

#### Mobile
- Stocks
image
image

- DAF 
image

### CODE REVIEW

File: src/components/IFrame.tsx [hunk 1]
```diff
@@ -17,12 +17,15 @@ export default function IFrame({
...
-        <div className={className}>
-          <Loader
-            bgColorClass="bg-blue dark:bg-white"
-            gapClass="gap-2"
-            widthClass="w-4"
-          />
+        <div
+          className={`${className} flex justify-center items-center gap-2`}
+          style={{
+            height: rest.height ? `${rest.height}px` : undefined,
```
Comment (misicnenad) [inline src/components/IFrame.tsx:23]: Sets the correct dimensions against which to center align `LoaderRing`

File: src/components/donation/Steps/DonateMethods/DonateMethods.tsx [hunk 2]
```diff
@@ -23,15 +25,19 @@ export default function DonateMethods({ donaterConfig, state }: Props) {
...
         <Tab className={({ selected }) => tabClasses(selected)}>
           Credit/Debit
         </Tab>
         <Tab className={({ selected }) => tabClasses(selected)}>Crypto</Tab>
-        <Tab className={({ selected }) => tabClasses(selected)}>Other</Tab>
+        <Tab className={({ selected }) => tabClasses(selected)}>Stocks</Tab>
+        <Tab className={({ selected }) => tabClasses(selected)}>DAF</Tab>
       </Tab.List>
       <Tab.Panels>
-        <Tab.Panel className="grid">
```
Comment (misicnenad) [inline src/components/donation/Steps/DonateMethods/DonateMethods.tsx:34]: redundant

File: src/components/donation/Steps/DonateMethods/DonateMethods.tsx [hunk 3]
```diff
@@ -23,15 +25,19 @@ export default function DonateMethods({ donaterConfig, state }: Props) {
       className="grid content-start mt-2"
       defaultIndex={state.details?.method === "crypto" ? 1 : 0}
     >
-      <Tab.List as="div" className="grid grid-cols-3 mb-6">
+      <Tab.List
+        as="div"
+        className="grid grid-flow-col sm:grid-cols-4 justify-stretch mb-6"
```
Comment (misicnenad) [inline src/components/donation/Steps/DonateMethods/DonateMethods.tsx]: Tried making the tab list have an auto-overflow with a scroller on mobile screens, but had difficulties. 
Opted for updating tab widths to fit the mobile screen width and the `grid-flow-col sm:grid-cols-4` was the only one that did the job.
@SovereignAndrey @ap-justin open to suggestions here

**Comment by misicnenad** [comment]
@SovereignAndrey @ap-justin 
current tab layout on mobile has tabs of different size (notice in image below that "DAF" tab is smaller than the rest):
image

an alternative tab layout that preserves tab width would be:
image

**Review by SovereignAndrey — APPROVED** [review]

**Pull request merged**